modules/@apostrophecms/modal/ui/apos/apps/AposModals.js

```diff
@@ -1,70 +1,22 @@
 import createApp from 'Modules/@apostrophecms/ui/lib/vue';
+import { useModalStore } from 'Modules/@apostrophecms/ui/stores/modal';
 
 export default function() {
   const component = apos.vueComponents.TheAposModals;
   const el = document.querySelector('#apos-modals');
   if (!el) {
     return;
   }
-  const app = createApp(component, {
-    modals: apos.modal.modals
-  });
-  const theAposModals = app.mount(el);
+  const app = createApp(component);
+  app.mount(el);
 
-  apos.modal.execute = theAposModals.execute;
-  apos.modal.getAt = theAposModals.getAt;
-  apos.modal.getProperties = theAposModals.getProperties;
-  apos.modal.onTopOf = onTopOf;
-  apos.modal.stack = [];
-  apos.confirm = theAposModals.confirm;
-  apos.alert = theAposModals.alert;
-}
+  const modalStore = useModalStore();
 
-// Returns true if el1 is "on top of" el2 in the
-// modal stack, as viewed by the user. If el1 is a
-// modal that appears later in the stack than el2
-// (visually stacked on top), this method returns true.
-// If el2 is `document` and el1 is a modal, this
-// method returns true. For convenenience, if el1
-// or el2 is not a modal, it is treated as its DOM
-// parent modal, or as `document`. If el1 has no
-// parent modal this method always returns false.
-//
-// If el1 is no longer connected to the DOM then it
-// is also considered to be "on top" e.g. not something
-// that should concern `v-click-outside-element` and
-// similar functionality. This is necessary because
-// sometimes Vue removes elements from the DOM before
-// we can examine their relationships.
-function onTopOf(el1, el2) {
```

Comment:
Method moved in store. Not sure it's used anymore since we use the overlay click to close a modal.

Reply:
See `click-outside-element.js`. Make sure that functionality does not break with this change.